src/solve.jl

```diff
@@ -1,8 +1,9 @@
 function init(prob::AbstractDDEProblem{uType,tType,lType,isinplace}, alg::algType,
               timeseries_init=uType[], ts_init=tType[], ks_init=[];
               d_discontinuities=tType[], dtmax=tType(7*minimum(prob.lags)), dt=zero(tType),
-              kwargs...) where {uType,tType,isinplace,
-                                algType<:AbstractMethodOfStepsAlgorithm,lType}
+              saveat=tType[], tstops=tType[], save_idxs=nothing,
```

Comment:
Building an output solution at the end? I'm not sure, I will try to compare it.
Another idea would be to add an option `calc_idxs` that already restricts all calculations to a subset of components and/or intermediate purging of the solution, since one probably only needs interpolation in the interval `[t-maximum(prob.lags), t]`.